classes/fields/date.php

```diff
@@ -166,6 +166,19 @@ public function schema( $options = null ) {
 		return $schema;
 	}
 
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$options" missing
* :no_entry_sign: Doc comment for parameter "$js" missing

<sub>hash: e2b2ae39138a14fef13fc4a0e02000d6</sub>